.github/workflows/update-and-publish-helm-chart.yml

```diff
@@ -0,0 +1,44 @@
+name: update-and-publish-cromwell-helm-chart
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+  update-and-publish:
+    name: Update and publish new cromwell-helm chart
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2
+
+      - name: Find out next chart version
+        run: |
+          currentVersion=$(grep 'version:' cromwell-helm/Chart.yaml | awk '{print $NF}')
+
+          # this will increment only the patch version digit. For example, it will update 0.1.10 => 0.1.11
+          nextVersion=$(echo ${currentVersion} | awk -F. -v OFS=. '{$NF += 1 ; print}')
+
+          echo "CURRENT_VERSION=$currentVersion" >> $GITHUB_ENV
+          echo "NEXT_VERSION=$nextVersion" >> $GITHUB_ENV
+
+      - name: Bump chart version
+        run: |
+          sed -i "s/$CURRENT_VERSION/$NEXT_VERSION/" cromwell-helm/Chart.yaml
+
+      - name: Publish the new helm chart and update index.yaml
+        run: |
+          helm package cromwell-helm/
+          mv cromwell-${NEXT_VERSION}.tgz charts/cromwell-${NEXT_VERSION}.tgz
+          helm repo index --url https://broadinstitute.github.io/cromwhelm/charts/ charts
+
+      - name: Push changes to GitHub
+        env:
+          BROADBOT_GITHUB_TOKEN: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
+        run: |
+          git checkout main
+          git diff
```

Comment:
What's this diff for?

Reply:
This is just to print the diff in the action logs. I thought it would good to have it printed in logs.

Reply:
Yeah I did the same, it's really handy to have an easy way to check changes